modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -1,35 +1,282 @@
+kubernetes   1.13.12
 ```
 
+[NOTE]
+The Runtime Fabric component and cluster-level component (appliance) versions that are in use are also displayed in the *Runtime Fabrics* tab in Runtime Manager. If an upgrade is available, an "Upgrade to vx.x" message is displayed.
+
+== Determine the Most Recent Component Versions Available
+The most recent component versions are displayed in the *Downloads* page, located in Runtime Manager in the *Runtime Fabrics* tab. A link to each package’s release notes is provided to show the changes.
+
+You can also review the xref:release-notes::runtime-fabric/runtime-fabric-release-notes.adoc[Anypoint Runtime Fabric Release Notes] information.
+
+=== Apply the Upgrade
+Before upgrading your production environment, perform the following steps on a nonproduction Runtime Fabric to verify compatibility with your environment.
+
+. Review the xref:release-notes::runtime-fabric/runtime-fabric-release-notes.adoc[Anypoint Runtime Fabric Release Notes] information.
+. Provision additional worker VMs as needed to maintain availability of applications:
+
+.. If the amount of available resources for application deployments is less than the resources of one worker node, add another worker VM with the same amount of CPU, memory, and disk resources to the Runtime Fabric.
++
+See xref:manage-nodes.adoc[How to Add a Node to Runtime Fabric].
+. To upgrade Runtime Fabric components to a later version:
+
+.. Plan to upgrade when your team is not deploying new applications or managing existing applications on Runtime Fabric.  
+.. Navigate to your Runtime Fabric by selecting it on the *Runtime Fabrics* tab in Runtime Manager.
+.. Select "Upgrade to vx.x". The status of the Runtime Fabric is updated to show that the upgrade procedure is in progress.
+.. Wait for the status to transition to *Active*. 
++
+In most cases, the upgrade takes less than 10 minutes. When the status transitions to *Active*, and the version of the Runtime Fabric reflects the new version, the upgrade is complete.
++
+If the Runtime Fabric and Runtime Manager connection disconnects during the upgrade, connectivity is restored after the upgrade finishes.
++
+Keep your browser session open to use in later steps.
++
 [IMPORTANT]
 To upgrade to Runtime Fabric version 1.5 or later, you must be running version 1.4.54. If you are not running version 1.4.54, upgrade to 1.4.54 before upgrading to 1.5.
+. If you are not using the latest version of the `rtfctl` command-line utility, download the latest version:
+.. Using your terminal on a controller node, run the following command: 
++
+----
+cd /opt/anypoint/runtimefabric
+sudo curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -o rtfctl
+----
++
+.. Change file permissions for the `rtfctl` binary: 
++
+----
+sudo chmod +x rtfctl
+----
+. If you are not using the latest version of the cluster-level components (installer package), upgrade to the latest version:
+
```

Comment:
Yes, this is all nested unnecessarily deeply. Please adjust all the following format tags to promote them one level per the previous edit. I will not retag these myself, to avoid assuming your intent.

Reply:
(Always review your organization if you get to three levels and beyond.)